example.js

```diff
@@ -0,0 +1,25 @@
+
+var Stagger = require('./lib/stagger');
+
+var stagger = Stagger.getDefault({
+  "pid": "1234",
+  "cmd": "./hello"
+});
+
+// function is called when report is requested
+stagger.registerCount('count0', function() {
+  return 1;
+});
+
+function go() {
+  setTimeout(function () {
```

Comment:
It's better to use setInterval in JavaScript to avoid drifting in time unless that's what you want. Eg: when you want to have back-pressure.